python/pyspark/sql/column.py

```diff
@@ -329,6 +329,8 @@ def cast(self, dataType):
         [Row(ages=u'2'), Row(ages=u'5')]
         >>> df.select(df.age.cast(StringType()).alias('ages')).collect()
         [Row(ages=u'2'), Row(ages=u'5')]
+        
+	(Note : astype is alias for cast)
```

Comment:
The problem that I was referring to is exactly that the documentation is repeated instead of being adapted for the function that is described.  The code examples for `astype` should use `astype` not `cast`.